### PR TITLE
keep same log format style consistent

### PR DESCRIPTION
**Pull request by cwdsuzhou**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/volume/csi/csi_attacher.go [hunk 1]
```diff
@@ -255,7 +255,7 @@ func (c *csiAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.No
 
 		skip, err := c.plugin.skipAttach(driverName)
 		if err != nil {
-			klog.Error(log("Failed to check CSIDriver for %s: %s", driverName, err))
```
Comment (jsafrane) [inline pkg/volume/csi/csi_attacher.go]: What is purpose of this PR? There is IMO no difference between printing `error` as `%v` and `%s`.

Comment (cwdsuzhou) [reply pkg/volume/csi/csi_attacher.go]: They have same results, just keep same log format style consistent, rather other than sometimes using `%s`, sometimes using `%v` for the same variable

Comment (jsafrane) [reply pkg/volume/csi/csi_attacher.go]: I think it's inevitable in a big project like this. AFAIK we don't have strict guidelines how `error` should be printed and different people prefer different style.

Comment (cwdsuzhou) [reply pkg/volume/csi/csi_attacher.go]: Yep， you are right. But I think it’s better to keep consistent in the same func.

**Pull request closed without merging**